packages/block-library/src/embed/edit.js

```diff
@@ -201,21 +221,18 @@ const EmbedEdit = ( props ) => {
 					icon={ icon }
 					label={ label }
 					onFocus={ onFocus }
-					onSubmit={ ( event ) => {
-						if ( event ) {
-							event.preventDefault();
-						}
-
-						setIsEditingURL( false );
-						setAttributes( { url } );
-					} }
+					onSubmit={ Platform.select( {
+						web: onSubmit,
+						native: onSubmitNative,
+					} ) }
```

Comment:
Although this change shouldn't break the logic of the web version, I think it's recommended to test it as a double-check.

Reply:
I verified the web GB using the web `onSubmit()` and only calling it once for that matter ✅